src/Store.js

```diff
@@ -470,14 +469,14 @@ class Store {
     }
   }
 
-  async _addOperation (data, batchOperation, lastOperation, onProgressCallback) {
+  async _addOperation (data, { batchOperation, lastOperation, onProgressCallback, pin = false } = {}) {
```

Comment:
This means the API changes and needs changes in all stores. How about instead adding `pin` as the last arg and keep the current API?

Reply:
@haadcode Seems to me all the stores need to be changed regardless in order to pass the options, no?

Reply:
To pass (use/enable) the option yes, but if they don't change anything, their old code still works as expected, whereas changing the API would require all dependants to change their `_addOperation()` calls to match the changed API.